java/anaplan-connect-library/ac-auth/ac-auth-default/src/test/resources/test.properties

```diff
@@ -0,0 +1,12 @@
+service.location=https://mock-api.anaplan.com
```

Comment:
Delete this file please

Reply:
This test properties has usages across tests. In this case should I delete it?